CHANGES.rst

```diff
@@ -15,23 +15,29 @@ development and backward compatability is not ensured.
 Major changes
 -------------
 
+* :class:`FeatureAugmenter` is renamed to :class:`Joiner`.
+  :pr:`674` by :user:`Jovan Stojanovic <jovan-stojanovic>`
+
 * :func:`fuzzy_join` and :class:`FeatureAugmenter` can now join on datetime columns.
   :pr:`552` by :user:`Jovan Stojanovic <jovan-stojanovic>`
 
+* :class:`Joiner` now supports many-to-many joins.
```

Comment:
I would not call this "many to many" but join keys defined across multiple columns